components/layout.js

```diff
@@ -34,9 +35,16 @@ export default function Layout({ children, home }) {
             </div>
           </header>
         )}
-        <div className={styles.container}>
-          <div>{children}</div>
-        </div>
+        {!myJobs ? (
+          <div className={styles.container}>
+            <div>{children}</div>
+          </div>
+        ) : (
+          <div className={styles.containerJobs}>
+            <div>{children}</div>
+          </div>
+        )}
```

Comment:
What do you think about it?

```suggestion
          <div className={ myJobs ? styles.containerJobs : styles.container}>
            <div>{children}</div>
          </div>
```